etsd/keys/templates/keys/publickey_detail.html

```diff
@@ -45,12 +21,18 @@ <h5 class="modal-title" id="confirmModalLabel">Public Key Confirmation</h5>
         The public key needs confirmation by the administration in order to be used. 
       </div>
       {% if perms.core.admin %}
-        <!-- Button trigger modal -->
-      <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#confirmModal">{% translate "Confirmation" %}</button>
+      <form action="{% url 'publickey_accept' publickey.id %}" method="POST" style="display:inline">
```

Comment:
I have this weird behavior. If a button uses confirmation with bootstrap5alert it strips the POST data of the button name. If a button goes straight to action it includes it in the POST data. 
Is it ok to set form value through javascript?

Reply:
I presume this is because the form is submited programatically with the form.submit() and not with click the button... So I guess you'll need to set the value through JS as you recommend.